js/term.js

```diff
@@ -2,117 +2,124 @@
  * @package Polylang
  */
 
-// quick edit
-(function( $ ) {
-	$( document ).bind(
-		'DOMNodeInserted',
-		function( e ) {
-			var t = $( e.target );
-
-			// WP inserts the quick edit from
-			if ( 'inline-edit' == t.attr( 'id' ) ) {
-				var term_id = t.prev().attr( 'id' ).replace( "tag-", "" );
-
-				if ( term_id > 0 ) {
-					// language dropdown
-					var select = t.find( ':input[name="inline_lang_choice"]' );
-					var lang = $( '#lang_' + term_id ).html();
-					select.val( lang ); // populates the dropdown
-
-					// disable the language dropdown for default categories
-					var default_cat = $( '#default_cat_' + term_id ).html();
-					if ( term_id == default_cat ) {
-						select.prop( 'disabled', true );
+/**
+ * Quick edit
+ */
+jQuery(
+	function( $ ) {
+		$( document ).on(
+			'DOMNodeInserted',
+			function( e ) {
```

Comment:
Same comment as PR #614 about DOMNodeInserted mutation events deprecation
https://github.com/polylang/polylang/pull/614#discussion_r516651633
Just to keep in mind.